docs/userGuide/cliCommands.md

```diff
@@ -203,9 +202,9 @@ The caveat is that not building all pages during the initial process, or not reb
 %%{{ icon_info }} Related: [User Guide: Deploying the Website](deployingTheSite.html).%%
 
 **{{ icon_examples }}**
-* `markbind deploy`
-* `markbind deploy ./myWebsite` : Generates and deploys the site from the `myWebsite` directory.
-* `markbind deploy --no-build` : Deploys the site without generating it first.
+* `markbind deploy` : Deploys after generating the site from the current working directory.
+* `markbind deploy ./myWebsite` : Deploys after generating the site from the `./myWebsite` directory.
+* `markbind deploy --no-build` : Deploys the site from the current working directory without generating the site.
```

Comment:
Just a personal opinion but perhaps the second "the site" could be replaced with "it" to make it sound more concise

Reply:
Fair enough! Updated in [2e73648](https://github.com/MarkBind/markbind/pull/2247/commits/2e73648dee577fcdc954b735e17e231648094dcb).